doc/quickstart.md

```diff
@@ -43,6 +43,11 @@ On Linux/macOS, you first have to follow the instructions printed by the Lean in
 Note: Packages **have** to be opened using "File > Open Folder..." for imports to work.
 Saved changes are visible in other files after running "Lean 4: Refresh File Dependencies" (`Ctrl+Shift+X`).
 
+## Abbreviations
+
+The VS Code editor helps you type nice Unicode characters supported by Lean,
```

Comment:
I don't want to have `abbreviations.json` in this repo. We can say users can use the VS Code command "Lean 4: Show all abbreviations" to list all abbreviations. We can also remind them that when hovering over Unicode symbols, VS Code will show how to input them.

Reply:
Ok, done.  The intro to abbreviations in the quickstart now says this:

![image](https://user-images.githubusercontent.com/18707114/189984574-fe852f1b-fd6e-482a-80cb-2431b4f42cfd.png)